resources/css/filament/customer/theme.css

```
@@ -1,8 +1,26 @@
-@import '../../../../vendor/filament/filament/resources/css/theme.css';
+@import 'tailwindcss';
 @plugin "@tailwindcss/typography";
-@source '../../../../vendor/guava/filament-knowledge-base/resources/**/*.blade.php';
+@import '../../../../vendor/filament/filament/resources/css/theme.css';
 
 @source '../../../../app/Filament/Customer/**/*';
 @source '../../../../resources/views/filament/customer/**/*';
 
+/* @source inline — ensures dynamic status badge classes survive purging */
+@source inline(
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)